app/Helpers/DatabaseHelper.php

```diff
@@ -89,35 +92,35 @@ public static function getCommentsMax($type){
         return $v->maxviews;
     }
 
-    public static function langId_from_short($short){
+    public static function langId_from_short($short) {
```

Comment:
__[code_sniffer]__
* Method name "DatabaseHelper::langId_from_short" is not in camel caps format
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->